manager/src/main/java/org/jmeterplugins/repository/PluginManager.java

```diff
@@ -331,4 +325,91 @@ private String getAllPluginsStatusString() {
         }
         return Arrays.toString(res.toArray());
     }
+
+    /**
+     * @return Available plugins
+     */
+    public static String getAvailablePluginsAsString() {
+        PluginManager manager = getStaticManager();
+        return manager.getAvailablePluginsString();
+    }
+
+    private String getAvailablePluginsString() {
+        final StringBuilder buf = new StringBuilder("Available Plugins\r\n");
+
+        for (Plugin plugin : getAvailablePlugins()) {
+            Set<String> versionSet = plugin.getVersions();
+            String[] versions = versionSet.toArray(new String[versionSet.size()]);
+
+            if (versions.length == 1) {
```

Comment:
Don't vary output for users, it will make any automation scripts crazy. Be lean.